pydantic_factories/value_generators/constrained_number.py

```diff
@@ -55,7 +56,7 @@ def get_constrained_number_range(
     minimum = get_value_or_none(equal_value=ge, constrained=gt, increment=get_increment(t_type))
     maximum = get_value_or_none(equal_value=le, constrained=lt, increment=-get_increment(t_type))  # pyright: ignore
 
-    if minimum is not None and maximum is not None and maximum <= minimum:
+    if minimum is not None and maximum is not None and maximum < minimum:
```

Comment:
It doesn't really make sense to test floats and decimals the same way I tested ints so here I just make sure the test won't fail accidentally.